textproto/header.go

```diff
@@ -87,9 +98,20 @@ func (h *Header) AddRaw(kv []byte) {
 	h.m[k] = append(h.m[k], f)
 }
 
+var headerNameRegex = regexp.MustCompile("^[!-9;-~]*$")
```

Comment:
That was a bit confusing to read at first. Perhaps a little commentary can help (e.g. "matches string of ASCII printable characters except for colon") or [just get rid of the regular expression here](https://xkcd.com/1171/) and replace it with a loop.

Reply:
You're 100% right here.
I hope now `':'` is highlighted more explicitly now.